hack/start-minikube.sh

```diff
@@ -9,6 +9,8 @@ MINIKUBE_KUBERNETES_VERSION="${MINIKUBE_KUBERNETES_VERSION:-}"
 MINIKUBE_CPUS="${MINIKUBE_CPUS:-}"
 MINIKUBE_MEMORY="${MINIKUBE_MEMORY:-}"
 MINIKUBE_CNI="${MINIKUBE_CNI:-}"
+LOCAL_MAVEN_CACHE="${LOCAL_MAVEN_CACHE:-}"
+MOUNTED_MAVEN_CACHE="${MOUNTED_MAVEN_CACHE:"/data/m2"}"
```

Comment:
`MOUNTED_MAVEN_CACHE="${MOUNTED_MAVEN_CACHE:"/data/m2"}"` should be:
`MOUNTED_MAVEN_CACHE="${MOUNTED_MAVEN_CACHE:-/data/m2}"` if I am not mistaken.